### ISSUE TITLE
MAS 1.4.11 - Non-text contrast

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#7528]
UI components (anything that conveys information) needs to have color contrast of at least 3:1.

For context, previously only text needed to have proper contrast. We are now required to make sure graphs and other UI elements have proper contrast as well.

**Comment by scottbommarito** [comment]
The statistics graph color on hover is not sufficiently contrasted with the background.

**Comment by scottbommarito** [comment]
The status page table background colors do not have enough contrast with the background.

**Comment by scottbommarito** [comment]
The GitHub stars icon does not have sufficient contrast.

**Comment by scottbommarito** [comment]
The RSS feed and twitter icons on the display package page do not have sufficient contrast.

**Comment by scottbommarito** [comment]
The API key legacy icon does not have sufficient contrast.
